cvat/apps/annotation/mask_instance.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (C) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+format_spec = {
```

Comment:
I would suggest adding this functionality to `pascal_voc.py` nearby as one extra dumper in the corresponding list.

Reply:
@zhiltsov-max , @gachiemchiep , It is a great idea to join several parsers. I believe MASK and MASK INSTANCE are very similar. Thus it is better to extend "MASK" annotation format instead of creating a new one. Thus it will not be necessary to duplicate code.

Reply:
@zhiltsov-max  your idea make sense. these masks are used for pascal voc so it better move into the same location.

Reply:
@zhiltsov-max you're right. mask and mask instance are used in pascal voc so those parser should be merged into pascal_voc.py. Maybe it is a good idea to parse "bounding box" + "mask" + "mask instance" at the same time.

Reply:
@zhiltsov-max I merged the mask_instance and mask into mask.py.